agent/provider/src/market/negotiator/builtin/price.rs

```diff
@@ -0,0 +1,72 @@
+use ya_agreement_utils::OfferDefinition;
+
+use crate::market::negotiator::factory::AgreementExpirationNegotiatorConfig;
+use crate::market::negotiator::{
+    AgreementResult, NegotiationResult, NegotiatorComponent, ProposalView,
+};
+
+pub struct PriceNego {}
+
+static PRICE_PROPERTY: &'static str = "/golem/com/pricing/model/linear/coeffs";
```

Comment:
This property doesn't follow negotiable property convention, which requires to have `?` on the end of property name.

Second thing: We change semantic of this property. How this affect compatibility?